Simplenote/src/main/res/values-ar/strings.xml

```diff
@@ -66,11 +84,11 @@ Language: ar
     <string name="toast_email_sent">تم إرسال رسالة عبر البريد الإلكتروني</string>
     <string name="fullscreen_verify_email_button_secondary">إعادة إرسال بريد إلكتروني</string>
     <string name="fullscreen_verify_email_title">التحقُّق من بريدك الإلكتروني</string>
-    <string name="fullscreen_verify_email_subtitle">أرسلنا رسالة بريد إلكتروني خاصة بالتحقق إلى &lt;b&gt;%1$s&lt;/b&gt;. يرجى التحقُّق من علبة الوارد لديك واتباع الإرشادات.</string>
+    <string name="fullscreen_verify_email_subtitle">أرسلنا رسالة بريد إلكتروني خاصة بالتحقق إلى &lt;b>%1$s&lt;/b>. يرجى التحقُّق من علبة الوارد لديك واتباع الإرشادات.</string>
```

Comment:
Hmm these look bad, could break the XML? @mokagio should we be using a script to generate these?